clients/src/test/java/org/apache/kafka/common/KafkaFutureTest.java

```diff
@@ -637,7 +637,12 @@ public void testLeakCompletableFuture() throws NoSuchMethodException, Invocation
 
             // Check the CF from a minimal CompletionStage doesn't cause completion of the original KafkaFuture
             Method minimal = CompletableFuture.class.getDeclaredMethod("minimalCompletionStage");
-            CompletionStage<String> cs = (CompletionStage<String>) minimal.invoke(comfut);
+            CompletionStage<String> cs = null;
+            try {
+                cs = (CompletionStage<String>) minimal.invoke(comfut);
+            } catch (InvocationTargetException e) {
+                throw e.getCause();
+            }
```

Comment:
Maybe we should have a helper `invoke` method since we do this many times in this test?

Reply:
Thanks for your suggestion. I have made the changes that you suggested in the latest commit.